website/content/v1.5/kubernetes-guides/configuration/deploy-metrics-server.md

```diff
@@ -21,6 +21,13 @@ machine:
       rotate-server-certificates: true
 ```
 
+This may require manually approving the kubelets certificate signing requests in the cluster PKI:
```

Comment:
this feels to be confusing a bit, or should be its own section, as the notes below describe how to install auto-approver.

I don't think a manual way is sustainable in the long run, but it might be some Note or special block which explains how to do it manually, but better to find automated solution

Reply:
You are totally right. I somehow managed to overlook the second part of the page, which completely solves this problem. Will close this PR.